lib/src/controllers/pod_player_controller.dart

```diff
@@ -15,6 +15,9 @@ class PodPlayerController {
   late String getTag;
   bool _isCtrInitialised = false;
 
+  //ignore: prefer_typing_uninitialized_variables
+  var _initializationError;
```

Comment:
Don't use `var` instead use `Exception? _initializationError ` or  `Object? _initializationError `
And remove ` //ignore: prefer_typing_uninitialized_variables`